docs/general/administration/migrate.md

```diff
@@ -5,9 +5,15 @@ title: Migrating
 
 # Migrating
 
-It is possible to migrate your system to another system by using environment variables.
-It's possible to do this via the command line or by using Docker environment variables.
-To read more, see the [Configuration](/docs/general/administration/configuration) page.
+This page explains how to migrate from Emby to Jellyfin as well as how to migrate a Jellyfin installation.
+
+Jellyfins internal databases are not designed for migrations. Depending on your case there may be ways to work around this, f.ex. by just migrating parts of the data, or because it's possible maintain the same file paths.
```

Comment:
The new database schema (jellyfin.db) is designed for migration, we just don't use it for everything yet. So this sentence is untrue. It also doesn't provide relevant information anyway.

Reply:
I don't want to trash any work here; I know that there are plans to establish a better database structure. However, to the people reading this page, IMO that's not relevant, because their data is still stuck in library.db and many other files with all of their flaws (w.r.t. to migration) - a.k.a. structures that were not designed with migrations in mind.  

So the key message I wanted to get across here is that except for specific cases, the user has no chance of successfully migrating a Jellyfin installation without data loss if he's not using a tool like mine. Or in other words, *the relevant information* is to answer the number one question people looking into this (probably) ask: "Can't I just copy it?"

That being said, maybe this is more accurate in your eyes and likely more clear for the user as well: 

> Jellyfins internal databases can't be copied nor adjusted easily. Depending...

---

<details>
<summary>About jellyfin.db...</summary>

Jellyfin.db requires search-and-replace patching as well because it stores the profile pictures with absolute paths (f.ex. `C:\ProgramData\Jellyfin\Server\config\users\Max\profile.jpg`). In that sense it's not better than the rest of the database. Once again, this might seem harsh/inaccurate from a developer perspective, but from a user perspective its simple: No, you can't copy the file. No, there is not an easy way to adjust it ("not easy" = "a text editor won't do it"). It's not been designed such that either one of these options were possible. 
Also, I don't know if you're familiar with [what I had to do](https://github.com/MMMZZZZ/Jellyfin-Migrator#technical-documentation) to get the migration working, but SQL editor and text editor are not sufficient. 

</details>

Reply:
Yeah noting that the database files cannot be copied without changes to them is worth mentioning.